consensus/istanbul/validator.go

```diff
@@ -80,11 +80,11 @@ type ValidatorSet interface {
 	// Return the demoted validator array
 	DemotedList() []Validator
 	// SubList composes a committee after setting a proposer with a default value.
-	SubList(prevHash common.Hash, view *View) []Validator
+	SubList(prevHash common.Hash, view *View, config *params.ChainConfig) []Validator
```

Comment:
@yoomee1313 How about passing `bool` isIstanbul instead of the whole chain configuration?

Reply:
@jeongkyun-oh Thanks for the comment, reflected at https://github.com/klaytn/klaytn/pull/980/commits/fae2cbdd7b64145e8dd8a6f16f8b111dd81e9a93. Please take a look.